docs/content/latest/drivers-orms/java/postgres-jdbc.md

```diff
@@ -0,0 +1,319 @@
+
+For example
+
+```sql
+CREATE TABLE IF NOT EXISTS employee (id int primary key, name varchar, age int, language text)
+```
+
+```java
+Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5433/yugabyte","yugabyte", "yugabyte");
+Statment stmt = conn.createStatement();
+try {
+  stm.execute("CREATE TABLE IF NOT EXISTS employee" +
+                    "  (id int primary key, name varchar, age int, language text)");
+
+} catch (SQLException e) {
+  System.err.println(e.getMessage());
+}
+```
+
+`java.sql.Statement` throws `java.sql.SQLException` exception which needs to handled in the java code. Read more on designing [Database schemas and tables](../../../../explore/ysql-language-features/databases-schemas-tables/).
+
+### Read and Write Data
+
+#### Insert Data
+
+In order to write data into YugabyteDB, execute the `INSERT` statement using `java.sql.Statement` interface.
+
+For example
+
+```java
+INSERT INTO employee VALUES (1, 'John', 35, 'Java')
+```
+
+```java
+Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5433/yugabyte","yugabyte", "yugabyte");
+Statment stmt = conn.createStatement();
+try {
+  String insertStr = "INSERT INTO employee VALUES (1, 'John', 35, 'Java')";
+  stmt.execute(insertStr);
+
+} catch (SQLException e) {
+  System.err.println(e.getMessage());
+}
+```
+
+For inserting data using JDBC clients, it is always a good pratice to use `java.sql.PreparedStatemet` for executing `INSERT` statements.
+
+```java
+Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5433/yugabyte","yugabyte", "yugabyte");
+Statment stmt = conn.createStatement();
+try {
+
+  PreparedStatement pstmt = connection.prepareStatement("INSERT INTO employees (id, name, age, langugage) VALUES (?, ?, ?, ?)");
+  pstmt.setInt(1, 1);
+  pstmt.setString(2, "John");
+  pstmt.setInt(3, 35);
+  pstmt.setString(4, "Java");
+  pstmt.execute();
+
+} catch (SQLException e) {
+  System.err.println(e.getMessage());
+}
+```
+
+#### Query Data
+
+In order to query data from YugabyteDB tables, execute the `SELECT` statement using `java.sql.statement` interface. Query results are returned using `java.sql.ResultSet` interface which can be iterated using `resultSet.next()` method for reading the data. Read more on [ResultSet](https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html).
+
+For example
+
+```sql
+SELECT * from employee;
+```
+
+```java
+Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5433/yugabyte","yugabyte", "yugabyte");
+Statment stmt = conn.createStatement();
+try {
+
+  ResultSet rs = stmt.executeQuery("SELECT * FROM employee");
+  while (rs.next()) {
+    System.out.println(String.format("Query returned: name = %s, age = %s, language = %s", rs.getString(2), rs.getString(3), rs.getString(4)));
+  }
+
+} catch (SQLException e) {
+  System.err.println(e.getMessage());
+}
+```
+
+### Configure SSL/TLS
```

Comment:
Belongs in Reference section? This page should just be the CRUD example?

Reply:
I removed the reference section details from this page. Added only the steps required for CRUD example.